### PR TITLE
Blockchain c1w10 - Cryptoeconomics

### PR DESCRIPTION
**Pull request by mihaiberq**
Covers #561 

- [x] Fix `*` to `-` changes made by prettier

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/blockchain/fundamentals/cryptoeconomics/how-do-cryptocurrencies-acquire-value.md?ref=1650c0bde9a0c7be6a1d497df5396691a3fc37dd
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/blockchain/fundamentals/cryptoeconomics/why-is-the-price-fluctuating-so-much.md?ref=1650c0bde9a0c7be6a1d497df5396691a3fc37dd

**Comment by enki-hq** [comment]
Rule insight:standards failed for files: 
  - blockchain/fundamentals/cryptoeconomics/bc-practice-market-cap.md
  - blockchain/fundamentals/cryptoeconomics/why-is-the-price-fluctuating-so-much.md
  - blockchain/fundamentals/cryptoeconomics/how-do-cryptocurrencies-acquire-value.md

**Comment by enki-hq** [comment]
Rule insight:exercise-data failed for files: 
  - blockchain/fundamentals/cryptoeconomics/bc-practice-market-cap.md

**Comment by enki-hq** [comment]
Rule insight:standards failed for files: 
  - blockchain/fundamentals/cryptoeconomics/bc-practice-market-cap.md
  - blockchain/fundamentals/cryptoeconomics/why-is-the-price-fluctuating-so-much.md
  - blockchain/fundamentals/cryptoeconomics/how-do-cryptocurrencies-acquire-value.md

**Comment by enki-hq** [comment]
Rule insight:standards failed for files: 
  - blockchain/fundamentals/cryptoeconomics/bc-practice-market-cap.md
  - blockchain/fundamentals/cryptoeconomics/how-do-cryptocurrencies-acquire-value.md
  - blockchain/fundamentals/cryptoeconomics/why-is-the-price-fluctuating-so-much.md

**Comment by enki-hq** [comment]
Rule insight:standards failed for files: 
  - blockchain/fundamentals/cryptoeconomics/bc-practice-market-cap.md
  - blockchain/fundamentals/cryptoeconomics/how-do-cryptocurrencies-acquire-value.md
  - blockchain/fundamentals/cryptoeconomics/why-is-the-price-fluctuating-so-much.md

File: blockchain/fundamentals/cryptoeconomics/bc-practice-market-cap.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+levels:
+
+  - beginner
+
+  - basic
+
+type: exercise
+
+link: https://coinmarketcap.com/historical/20180610/
```
[the remark this answers is not in the document]
Comment (mihaiberq) [reply blockchain/fundamentals/cryptoeconomics/bc-practice-market-cap.md]: sure

File: blockchain/fundamentals/cryptoeconomics/bc-practice-market-cap.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+linkType: coinmarketcap
+answer: 22
```
[the remark this answers is not in the document]
Comment (mihaiberq) [reply blockchain/fundamentals/cryptoeconomics/bc-practice-market-cap.md]: not really, it's the snapshot of a previous date, I wanted users to become accustomed to coinmarketcap and to be able to read stats

File: blockchain/fundamentals/cryptoeconomics/how-do-cryptocurrencies-acquire-value.md [hunk 3]
```diff
@@ -11,40 +10,51 @@ type: normal
...
+
 # How Do Cryptocurrencies Acquire Value?
 
 ---
+
 ## Content
 
+In general, how do things acquire value? The answer is straight-forward: they either serve a purpose/solve a problem (PC, cars, etc.) or enough people consider them to be valuable (currencies). The dollar/british pound/yen are useful because they can be used to buy food, a house or a car. Only humans and other living beings are intrinsically valuable since they can create value.
+
+Money is a concept created by us. It was simpler to trade a chicken now for some coins then trade the coins for wheat whenever needed. A piece of paper on which you write $10 has no value, nor the one on which you write $100. Objectively, the $10 or $100 inscribed on the banknote shouldn't make any difference. But because we recognize some banknotes as valuable, it is a difference worth $90.
```
[the remark this answers is not in the document]
Comment (mihaiberq) [reply blockchain/fundamentals/cryptoeconomics/how-do-cryptocurrencies-acquire-value.md]: indeed, updated it

File: blockchain/fundamentals/cryptoeconomics/how-do-cryptocurrencies-acquire-value.md [hunk 4]
```diff
@@ -11,40 +10,51 @@ type: normal
...
 
+That's money in general. With cryptocurrencies, the question is how much we value their attributes: ease of trade, proof of property, anonimity or the platform they represent. Bitcoin can be transferred anywhere globally, in a secure manner and relatively fast. The number of products for which a bitcoin can be exchanged also influences its value. However, its usage is limited to being an intermediary. Ways to extend the basic capabilities of Bitcoin are being researched but everything has to be adopted by means of consensus.
+
+Ether, on the other hand, powers the Ethereum network's smart contract system. On top of what Bitcoin does, Ethereum can also execute code files in exchange for gas. Gas can only be bought with ether (or rather its smallest unit, wei). As such, ether is used to pay for every transaction in order for it to be registered on the network. So, as long as you need/value what the Ethereum network can do, ether will also be valuable for you.
```
[the remark this answers is not in the document]
Comment (mihaiberq) [reply blockchain/fundamentals/cryptoeconomics/how-do-cryptocurrencies-acquire-value.md]: a new transaction is created for every function call. once it's included in a block (registered), it is also executed by the nodes. in a way, they're pretty much the same action.

ill add the same note to clarify to the insight

File: blockchain/fundamentals/cryptoeconomics/why-is-the-price-fluctuating-so-much.md [hunk 5]
```diff
@@ -11,20 +10,33 @@ type: normal
...
-# Why Is the Price Fluctuating so Much?
 
+# Why Is the Price Fluctuating so Much?
 
 ---
+
 ## Content
 
-Dummy content.
+In terms of the actual price (a coin's value in a certain fiat currency), no one can predict it exactly: who would have realistically thought that a bitcoin would reach $20,000?
```
[the remark this answers is not in the document]
Comment (mihaiberq) [reply blockchain/fundamentals/cryptoeconomics/why-is-the-price-fluctuating-so-much.md]: A fiat currency a currency backed by a government.

will reformulate the question

File: blockchain/fundamentals/cryptoeconomics/why-is-the-price-fluctuating-so-much.md [hunk 6]
```diff
@@ -11,20 +10,33 @@ type: normal
...
+
+In total, the market cap in December 2017 approached 1 trillion. Bitcoin's market cap never exceded $320 billions. Bitcoin traded volume was way smaller: maybe $20 billions on all exchanges corroborated, during peaks. In other words, it would be easy for someone that owns a relevant % of the total number of coins to manipulate the price in either directions. Bears drive the price down in order for them to accumulate, while bulls strive to move the price up for them to sell.
```
[the remark this answers is not in the document]
Comment (mihaiberq) [reply blockchain/fundamentals/cryptoeconomics/why-is-the-price-fluctuating-so-much.md]: I agree, this is a bit too much and the steep ending does not help. 

Wrt to market cap being explained - I've discussed it previously for bitcoin specifically. This is also the case for most terms.

File: blockchain/fundamentals/cryptoeconomics/why-is-the-price-fluctuating-so-much.md [hunk 7]
```diff
@@ -11,20 +10,33 @@ type: normal
...
+
+It's hard to have stability when most cryptocurrencies have no back. Only lately had platforms started to appear (read ERC20 tokens vs Bitcoin forks) for which their coins can be exchanged for a service, hence creating their own small economy. This is similar to how forex markets (where currencies are traded for one another) represent the economy of the countries: USD as the economy of USA and GBP as the economy of Great Britain.
+
+Moreover, there are no regulations: pump and dumps, inside trading or other (normally illegal) methods of price manipulation/prediction are not punished.
+
+Of course, markets in general cannot be explained in 5 minutes and many other things can be said about how cryptocurrency markets work. Given the how broad and unpredictable the topic is, self-study is required in order to understand all the mechanisms.
 
 ---
+
 ## Practice
```
[the remark this answers is not in the document]
Comment (mihaiberq) [reply blockchain/fundamentals/cryptoeconomics/why-is-the-price-fluctuating-so-much.md]: there should be a RQ:
```
What is the metric that describes how valuable a is cryptocurrency?

???

* Market cap
* Circulating supply
* Node investment
* Volume
```
Sure thing

**Comment by enki-hq** [comment]
Rule insight:standards failed for files: 
  - blockchain/fundamentals/cryptoeconomics/bc-practice-market-cap.md
  - blockchain/fundamentals/cryptoeconomics/why-is-the-price-fluctuating-so-much.md
  - blockchain/fundamentals/cryptoeconomics/how-do-cryptocurrencies-acquire-value.md

**Pull request merged**